Forta-Bots-Challenge-1/src/agent.ts

```diff
@@ -0,0 +1,83 @@
+import { Finding, HandleTransaction, TransactionEvent, FindingSeverity, FindingType } from "forta-agent";
+import {
+  CREATE_AGENT_SIGNATURE,
+  UPDATE_AGENT_SIGNATURE,
+  FORTA_REGISTRY_ADDRESS,
+  NETHERMIND_FORTA_DEPLOYER_ADDRESS,
+} from "./constants";
+
+export const provideHandleTransaction = (
+  createAgentSignature: string,
+  updateAgentSignature: string,
+  nethermindDeployAddress: string,
+  fortaRegistryAddress: string
+): HandleTransaction => {
+  return async (txEvent: TransactionEvent) => {
+    const findings: Finding[] = [];
+
+    // return empty findings if TransactionEvent is not from Nethermind
+    if (txEvent.from !== nethermindDeployAddress.toLowerCase()) {
+      return findings;
+    }
+
+    // filter TransactionEvent for bot deployments to Forta registry
+    const deploymentCalls = txEvent.filterFunction(createAgentSignature, fortaRegistryAddress);
+
+    // filter TransactionEvent for bot updates on Forta registry
+    const updateCalls = txEvent.filterFunction(updateAgentSignature, fortaRegistryAddress);
```

Comment:
```suggestion
    const functionCalls = txEvent.filterFunction([createAgentSignature, updateAgentSignature], fortaRegistryAddress);
```
You can take a look [here](https://github.com/forta-network/forta-bot-sdk/blob/master/sdk/transaction.event.ts#L102) to see that the `filterFunction` function takes `string[]`, so you could combine these.